app/routes/projects/$projectId/edit.tsx

```diff
@@ -156,8 +157,18 @@ export default function EditProjectPage() {
                 onChange={handleTabChange}
                 aria-label="Edit project"
               >
-                <TabStyles label="Project Details" />
-                <TabStyles label="Contributor's Path" />
+                {/* <TabStyles label="Project Details" />
```

Comment:
Maybe the commented code will be trigger an alert on sonarqube, can you remove the commented code ?